planning/behavior_velocity_planner/src/utilization/util.cpp

```diff
@@ -244,6 +244,78 @@ double calcJudgeLineDistWithJerkLimit(
   return std::max(0.0, x1 + x2 + x3);
 }
 
+double findReachTime(
+  const double j, const double a, const double v, const double d, const double min,
+  const double max)
+{
+  auto f = [](const double t, const double j, const double a, const double v, const double d) {
+    return j * t * t * t / 6.0 + a * t * t / 2.0 + v * t - d;
+  };
+  const double eps = 1e-5;
+  const int max_iter = 100;
+  double lower = min;
```

Comment:
TODO add 
add logic error not in this case
f(min) <0 
f(max)>0

Reply:
[5950d2e](https://github.com/autowarefoundation/autoware.universe/pull/388/commits/5950d2e8b0a757c72f6a3ebc553d12aaf47ce503)